packages/flutter/lib/src/material/selectable_text.dart

```diff
@@ -565,7 +571,7 @@ class _SelectableTextState extends State<SelectableText> with AutomaticKeepAlive
     final TextSelectionThemeData selectionTheme = TextSelectionTheme.of(context);
     final FocusNode focusNode = _effectiveFocusNode;
 
-    final TextSelectionControls textSelectionControls;
+    TextSelectionControls? textSelectionControls =  widget.textSelectionControls;
```

Comment:
Could this be `late`?  I'm still getting the hang of the nnbd stuff.

Reply:
It wouldn't make any change. The late keyword allows postponing the initializing of a non-nullable variable. But in that line, it is being initialized already with a variable that could be null.

Notice that EditableText accepts null in the `selectionControls` param.

If non-nullable is prefered we could leave that line like before and inside the EditableText constructor use `widget.textSelectionControls ?? textSelectionControls`